packages/editor/src/validators/content/index.test.ts

```diff
@@ -2,149 +2,171 @@ import { describe, expect, it, vi } from 'vitest';
     });
-    const mapArg = callback.mock.calls[0][0];
-    expect(mapArg).toBeInstanceOf(Map);
-    expect(mapArg.get('node-should-not-be-empty_5').tipPayload.nodeType).toBe('paragraph');
-    expect(mapArg.get('node-should-not-be-empty_13').tipPayload.nodeType).toBe('listItem');
-    expect(mapArg.get('node-should-not-be-empty_36').tipPayload.nodeType).toBe('definitionTerm');
-    expect(mapArg.get('node-should-not-be-empty_38').tipPayload.nodeType).toBe('definitionDescription');
   });
 
-  it('should notify editor of empty nodes th and td', async () => {
-    const callback = vi.fn();
-    await createTestEditor(
-      `
+  describe('Table validations', () => {
+    it('should notify editor of empty nodes th and td', async () => {
+      const callback = vi.fn();
+      await createTestEditor(
+        `
     <h1>foo</h1><table><tr><th>test</th><th></th></tr><tr><td> </td></td></tr></table>`,
-      callback,
-    );
+        callback,
+      );
+
+      await vi.waitFor(() => {
+        expect(callback).toHaveBeenCalledTimes(1);
+      });
+      const mapArg = callback.mock.calls[0][0];
+      expect(mapArg.get('node-should-not-be-empty_16').tipPayload.nodeType).toBe('tableHeader');
+      expect(mapArg.get('node-should-not-be-empty_22').tipPayload.nodeType).toBe('tableCell');
+    });
 
-    await vi.waitFor(() => {
-      expect(callback).toHaveBeenCalledTimes(1);
+    it('returns an empty caption warning', async () => {
+      const callback = vi.fn();
+      await createTestEditor(
+        `
+        <h1>Title</h1>
+        <table>
+          <caption></caption>
+          <thead>
+            <tr>
+              <th>tablehead</th>
+              <th>tablehead</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>table body cell</td>
+              <td>table body cell</td>
+            </tr>
+          </tbody>
+        </table>`,
+        callback,
+      );
+      expect(callback.mock.calls[0][0].get('node-should-not-be-empty_8').tipPayload.nodeType).toBe('tableCaption');
     });
-    const mapArg = callback.mock.calls[0][0];
-    expect(mapArg).toBeInstanceOf(Map);
-    expect(mapArg.get('node-should-not-be-empty_15').tipPayload.nodeType).toBe('tableHeader');
-    expect(mapArg.get('node-should-not-be-empty_21').tipPayload.nodeType).toBe('tableCell');
   });
 
-  it('should notify editor of empty link', async () => {
-    const callback = vi.fn();
-    await createTestEditor(
-      `
-    <h1>foo</h1><p>test with an empty<a href="https://example.com"> </a>link</p>`,
-      callback,
-    );
-
-    await vi.waitFor(() => {
-      expect(callback).toHaveBeenCalledTimes(1);
+  describe('Empty nodes', () => {
+    it('should notify editor of empty nodes p, li, dt and dd', async () => {
+      const callback = vi.fn();
+      await createTestEditor(
+        `
+    <h1>foo</h1><p></p><ul><li>foo</li><li> </li></ul><dl><dt>Term</dt><dd>Description</dd><dt></dt><dd></dd></dl>`,
```

Comment:
is er een reden dat de tags allemaal op 1 regel staan? Test file hierboven heeft alles wel keurig geformat, dat leest wat fijner 😆